packages/web/config/chain-infos.ts

```diff
@@ -903,12 +893,7 @@ const chainInfos = (
         average: 0.03,
         high: 0.05,
       },
-      features: [
-        "ibc-transfer",
-        "ibc-go",
-        "wasmd_0.24+",
-        "cosmwasm",
-      ],
+      features: ["ibc-transfer", "ibc-go", "wasmd_0.24+", "cosmwasm"],
```

Comment:
Hmm.. did your prettier fix do this? Is it looking at the right config? I think our config comes from next.js

Reply:
Yes, I used the cli directly